packages/demo/src/app/post-sample/components/icons/svg-icons-demo/svg-icons-demo.component.ts

```diff
@@ -7,7 +7,7 @@ import {Component, OnInit} from '@angular/core';
 })
 export class SvgIconsDemoComponent {
   packageName : string = "@swisspost/design-system-styles";
-  cwfImport : string = `@use "${this.packageName}/cwf";`;
+  cwfImport : string = `@use "${this.packageName}/core" as post;`;
```

Comment:
Maybe we should rename `cwfImport` as well?!

Reply:
This is part of an old migration documentation that will be removed anyways with #213.